ingestion/dbt_manifest_utils.py

```diff
@@ -62,7 +62,10 @@ def convert_cadet_manifest_table_to_datahub(node_info: dict) -> Tuple[str, str]:
     """
     domain = format_domain_name(node_info.get("fqn", [])[1])
     node_table_name = node_info.get("fqn", [])[-1]
-
+    # schema has the name of the database that has resulted from the filename parsing
+    database_name = node_info.get("schema", "")
+    if database_name.endswith("dev_dbt"):
+        node_table_name = f"{database_name}.{node_table_name.split('__')[-1]}"
```

Comment:
Here you have a special case for dev_dbt, and then on the line below is the old implementation that replaces double underscore with dots.

But in assign_cadet_databases you have a different expression (which I think does the same thing) - `node_table_name_no_double_underscore = f"{database}.{parts[-1]}"`

Can we make these use a common implementation?

Reply:
this was a leftover from me trying stuff out with the ingestion tests. Suffering from ingestion fatigue methinks - will improve